src/main/java/Duke.java

```diff
@@ -37,342 +48,367 @@ public void run() throws DukeException {
+            if (numTaskDone > inputs.size()) {
+                throw new DukeException("OOPS!!! Task does not exist.");
             }
+            System.out.println("Nice! I've marked this task as done:");
+            Input inputType = inputs.get(numTaskDone - 1);
+            inputType.taskDone();
+            System.out.println("[/] " + inputType.content + " " + inputType.printTime);
+        }
 
-            void taskDeadline(String nextLine) throws DukeException {
-                if (nextLine.equals("deadline") || nextLine.equals("deadline ")) {
-                    throw new DukeException("OOPS!!! The description of a deadline cannot be empty.");
-                }
-                int charLoc = nextLine.indexOf("/by");
-                Deadline deadline = new Deadline(nextLine.substring(9, charLoc), nextLine.substring(charLoc + 4));
-                inputs.add(deadline);
-                int count = inputs.size();
-                System.out.println("Got it. I've added this task: \n" + "  [D][x] " + deadline.content +
-                        deadline.printedTime + "\n Now you have " + count + " tasks in the list");
-                storage.writeToFile(inputs);
+        /*
+         * removes task, as specified by the user, from list
+         *
+         * @param nextLine, represents user input
+         * @throws DukeException if user does not specify which task to remove or task is not in list
+         */
+        void taskRemove(String nextLine) throws DukeException {
+            if (nextLine.equals("remove") || nextLine.equals("remove ")) {
+                throw new DukeException("OOPS!!! The description of remove cannot be empty");
             }
-
-            void taskEvent(String nextLine) throws DukeException {
-                if (nextLine.equals("event") || nextLine.equals("event ")) {
-                    throw new DukeException("OOPS!!! The description of an event cannot be empty.");
-                }
-                int charLoc = nextLine.indexOf("/at");
-                Event event = new Event(nextLine.substring(6, charLoc), nextLine.substring(charLoc + 4));
-                inputs.add(event);
-                int count = inputs.size();
-                System.out.println("Got it. I've added this task: \n" + "  [E][x] " + event.content +
-                        event.printedTime + "\n Now you have " + count + " tasks in the list");
-                storage.writeToFile(inputs);
+            int numTaskDone = Integer.valueOf(nextLine.substring(7));
+            if (numTaskDone > inputs.size()) {
+                throw new DukeException("OOPS!!! Task does not exist.");
             }
+            System.out.println("Noted. I've removed this task:");
+            Input inputType = inputs.get(numTaskDone - 1);
+            if (inputType.done) {
+                System.out.println("  " + inputType.id + "[/] " + inputType.content + inputType.printTime);
```

Comment:
You could try using the tick symbol instead of a '/'. The four-digit hex code for the tick symbol is \u2713.